workers/unity/Packages/com.improbable.gdk.buildsystem/Util/PathUtils.cs

```diff
@@ -0,0 +1,33 @@
+using System.IO;
+using System.Linq;
+
+namespace Improbable.Gdk.BuildSystem
+{
+    /// <summary>
+    ///     Contains common directories related to building assets and players.
+    /// </summary>
+    /// <remarks>
+    ///     All directories should be in Unity path format e.g. "Foo/Bar".
+    /// </remarks>
+    public static class PathUtils
+    {
+        public static readonly string BuildScratchDirectory = Path.GetFullPath(Path.Combine("build", "worker"));
+        public static readonly string AssetDatabaseDirectory = Combine("..", "..", "build", "assembly").ToUnityPath();
+        
+        /// <summary>
+        ///     Combines multiple path components via <c>Path.Combine</c>.
+        /// </summary>
+        internal static string Combine(params string[] paths)
```

Comment:
This isn't needed any more now that we're on a modern .NET version (4.x+ added the `params` overload of `Path.Combine`)